libraries/SRV_Channel/SRV_Channel.cpp

```diff
@@ -122,13 +131,18 @@ void SRV_Channel::calc_pwm(int16_t output_scaled)
     } else {
         pwm = pwm_from_range(output_scaled);
     }
+    // If not in unlimited ouput we contrain the ouput
+    if (unlimited_pwm_mask & (1U << ch_num)) {
```

Comment:
the variable name and the comment at initilization are confusing... or I miss read but for me the mask is set when at 0